src/jquery.uls.core.js

```diff
@@ -225,12 +225,7 @@
 			} );
 
 			// Handle key press events on the menu
-			this.$menu.on( 'keypress', $.proxy( this.keypress, this ) )
-				.on( 'keyup', $.proxy( this.keyup, this ) );
-
-			if ( this.eventSupported( 'keydown' ) ) {
-				this.$menu.on( 'keydown', $.proxy( this.keypress, this ) );
-			}
+			this.$menu.keydown( $.proxy( this.keypress, this ) );
```

Comment:
People seem to prefer `$elem.on( 'event', ... )` these day, maybe revert back to that form?

Reply:
done.